### PR TITLE
{bio}[foss/2017b] MaSuRCA 3.2.2 toolchain bump (REVIEW)

### PR DESCRIPTION
**Pull request by hajgato**

### CODE REVIEW

File: easybuild/easyconfigs/m/MaSuRCA/MaSuRCA-3.2.2-foss-2017b-Perl-5.26.0.eb [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+description = '''MaSuRCA is whole genome assembly software. It combines the efficiency of the de Bruijn graph  
+ and Overlap-Layout-Consensus (OLC) approaches. MaSuRCA can assemble data sets containing 
+ only short reads from Illumina sequencing or a mixture of short reads and long reads 
+ (Sanger, 454, Pacbio and Nanopore).'''
+
+toolchain = {'name': 'foss', 'version': '2017b'}
+
+# need a temporary url to download it. Do it manually here:
+# http://www.genome.umd.edu/masurca_form.html
```
Comment (migueldiascosta) [inline easybuild/easyconfigs/m/MaSuRCA/MaSuRCA-3.2.2-foss-2017b-Perl-5.26.0.eb]: is `3.2.2` version still easily available? From `3.2.4` it seems to be on github (https://github.com/alekseyzimin/masurca/releases), perhaps bump version and fetch from there?

Comment (hajgato) [reply easybuild/easyconfigs/m/MaSuRCA/MaSuRCA-3.2.2-foss-2017b-Perl-5.26.0.eb]: @migueldiascosta I needed that specific version, requested by an user. Bumping helps to get easyer the source, but then that should be a separate PR.

Comment (migueldiascosta) [reply easybuild/easyconfigs/m/MaSuRCA/MaSuRCA-3.2.2-foss-2017b-Perl-5.26.0.eb]: @hajgato ok - can you submit a test report for `3.2.2` then?

**Comment by hajgato** [comment]
Test report by @hajgato
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
nic160 - Linux centos linux 7.5.1804, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/8cd064f240ca34377350e547c1ddb5f7 for a full test report.

**Comment by hajgato** [comment]
@migueldiascosta Done!

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2415.golett.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/79029a23255d9c8f0242bdecd4397721 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
gligar04.gastly.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) Gold 6130 CPU @ 2.10GHz, Python 2.7.5
See https://gist.github.com/d34b1a1d32355b4f881de36f7088493f for a full test report.

**Comment by boegel** [comment]
Going in, thanks @hajgato!

**Pull request merged**